browser/app/js/components/Login.js

```diff
@@ -54,37 +49,27 @@ export class Login extends React.Component {
         history.push(minioBrowserPrefix)
       })
       .catch(e => {
-        dispatch(
-          actionsAlert.set({
-            type: "danger",
-            message: e.message
-          })
-        )
+        showAlert("danger", e.message)
       })
   }
 
   componentWillMount() {
-    const { dispatch } = this.props
+    const { clearAlert } = this.props
     // Clear out any stale message in the alert of previous page
-    dispatch(actionsAlert.clear())
+    clearAlert()
     document.body.classList.add("is-guest")
   }
 
   componentWillUnmount() {
     document.body.classList.remove("is-guest")
   }
 
-  clearAlert() {
-    const { dispatch } = this.props
-    dispatch(actionsAlert.clear())
-  }
-
   render() {
-    const { alert } = this.props
+    const { clearAlert, alert } = this.props
     if (web.LoggedIn()) {
       return <Redirect to={minioBrowserPrefix} />
     }
-    let alertBox = <Alert {...alert} onDismiss={this.clearAlert.bind(this)} />
+    let alertBox = <Alert {...alert} onDismiss={clearAlert.bind(this)} />
```

Comment:
`bind(this)` is not required, `onDismiss={clearAlert}` should work fine.

Reply:
@kmkanagaraj, thanks for the feedback, I have updated the commit accordingly. Also, I was going to ask if I should remove the contents inside the directory `browser/app/js/jest/__mocks__/` and change `package.json` back to its original version, as it looks like you are handling the errors on unit tests due to .svg files by downloading the identity-obj-proxy in PR #5499.

Reply:
@kaankabalak yes, while rebasing this PR you revert those files.